### ISSUE TITLE
contrib/rate_group fails to load (errors with mismatched brackets)

### ISSUE THREAD

**Issue by fuzzah** [darktable-org/lua-scripts#506]
Hello!

Sadly, the rate_group script fails to load.
Here's the log of `darktable -d lua`:
```
24.2628 LUA ERROR: dtutils.lua: prequire: 223: Error loading contrib/rate_group 
24.2628 LUA ERROR: dtutils.lua: prequire: 224: Error returned is error loading module 'contrib/rate_group' from file '/home/user/.config/darktable/lua/contrib/rate_group.lua':
    /home/user/.config/darktable/lua/contrib/rate_group.lua:106: ')' expected (to close '(' at line 103) near 'group' 
24.2629 LUA ERROR: script_manager.lua: activate: 510: error loading contrib/rate_group 
24.2629 LUA ERROR: script_manager.lua: activate: 511: error message: error loading module 'contrib/rate_group' from file '/home/user/.config/darktable/lua/contrib/rate_group.lua':
    /home/user/.config/darktable/lua/contrib/rate_group.lua:106: ')' expected (to close '(' at line 103) near 'group'
```

OS: Arch
Darktable was installed with pacman, the version is 4.8.1
The lua version is 5.4.7

How to troubleshoot this further?

**Comment by wpferguson** [comment]
Fixed by 8318569cdf51af5248d01251e0c29c5c3ed8dc94.  

Starting darktable should download the fixed script.

**Comment by fuzzah** [comment]
Thank you for the fix, it works now.
